app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -176,25 +176,29 @@ def parse_openshift_image(openshift_image)
         ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id
```

Comment:
I am not sure what is the meaning of an image without a name... also `parse_container_image` might return weird results.
Maybe we should find a way to ignore this "image" if that happens?